tests/test_lab_dev/test_circuits.py

```diff
@@ -314,9 +285,9 @@ def test_path_is_loaded(self, tmpdir):
         bs12 = BSgate([1, 2])
 
         circ = Circuit([vac, s01, bs01, bs12])
-        assert not load(circ.serialize())._path
+        assert load(circ.serialize()).path
 
-        circ.make_path()
-        assert circ._path
+        circ.optimize()
+        assert circ.path  # optimized path
 
-        assert load(circ.serialize())._path == circ.path
+        assert load(circ.serialize()).path == circ.path
```

Comment:
I can't comment a suggestion on a diff like this, but I'd rewrite this test as follows:
```python
circ = Circuit([vac, s01, bs01, bs12])
base_path = circ.path
assert load(circ.serialize()).path == base_path

circ.optimize()
opt_path = circ.path
assert opt_path != base_path
assert load(circ.serialize()).path == opt_path
```